clusterloader2/docs/experiments.md

```diff
@@ -0,0 +1,101 @@
+# Clusterloader2 - experiment rollout
+
+In this doc any change to the behavior of clusterloader2 is referred as an
+"experiment".
+
+## Motivation
+
+clusterloader2 is a tool used by all scalability, performance tests. Tests
+compile clusterloader2 at [HEAD], thus introducing breaking changes to
+clusterloader2 will stop scalability tests from passing at all. They have a large
+blast radius: every PR to k/k needs to pass. Also while for smaller and faster
+tests, breakages aren't that costly (unless they happen on the weekend, see
+https://github.com/kubernetes/perf-tests/pull/586), they are expensive for
+large, rarely run tests (e.g. [ci-kubernetes-e2e-gce-scale-performance]).
+
+For this reason, all new changes/features added to clusterloader2 shall be gated
+and rolled out gradually. This way, we can minimize blast radius of breaking
+changes. We can even stop some kinds of them to happen at all, as they should be
+caught at presubmit time and not allowed to merge at all.
+
+### General principles
+
+1. Change to clusterloader2 behavior is understood as:
+
+   - enabling new measurement
+   - changing semantic of an existing measurement
+   - changing how clusterloader2 setups cluster and run tests
```

Comment:
Ohh - I see it's here - I would suggest moving it to the beginning and say that this is what we later call experiment.